xml/System.Data.SqlClient/SqlDataReader.xml

```diff
@@ -246,9 +246,17 @@
         <Parameter Name="disposing" Type="System.Boolean" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="disposing">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="disposing"><see langword="true" /> to release managed and unmanaged resources; <see langword="false" /> to release only unmanaged resources.</param>
+        <summary>Releases the managed resources used by the <see cref="T:System.Data.SqlClient.SqlDataReader" /> and optionally releases the unmanaged resources.</summary>
```

Comment:
LGTM - even though SqlDataReader doesn't actually seem to have unmanaged resources. This is the standard dispose pattern text and we should probably keep it.